### ISSUE TITLE
'Clipper' exceeds bounds of netcdf tile?

### ISSUE THREAD

**Issue by CAP-GIS** [SCGeology/netcdf2ascii#2]
I understand the project at hand may not extend beyond a single Daymet tile. If I remember correctly, there are about 5 Daymet tiles covering SC, and perhaps 9 covering the Greater SC Drainage Area. These scripts could have broader applicability if we extended them to include merging Daymet tiles.

If we like Daymet data, and we plan to use it in a variety of contexts, we might want to go ahead and derive a dataset based on Daymet in a format more suitable to our uses. This could include pre-processing steps such as merging, reprojecting, and saving in a suitable format to our servers. Then the user can extract pieces as needed.

**Comment by freewheelCarto** [comment]
Yeah, if there is a use for this that is beyond the specific recharge model this was develop for, then we could definitely extend it to be more inclusive of different options. Maybe just a set of daymet/metdata processing tools that do different things.

The daymet and metdata files that we are using as input are already merged, nationwide coverage. Otherwise, we'd have to merge, then clip.
